rfd/0223-install-docs.md

```diff
@@ -0,0 +1,194 @@
+---
+authors: Stephen Levine (stephen.levine@goteleport.com)
+state: draft
+---
+
+# RFD 223 - Installation and Upgrade Documentation Organization
+
+# Required Approvers
+* Engineering: @klizhentas || @r0mant
+* Product / Design: @roraback && @ptgott
+
+## What
+
+When users get started with Teleport, they often look for pages of documentation related to downloading and installing
+a Teleport component that they need to use.
+
+This RFD proposes that we reorganize our documentation to make it easier to find these pages.
+
+## Why
+
+Currently, it can be difficult for users to find this documentation because it lives in different places depending
+on the component, target OS/platform, and other factors.
+
+For example, a common user workflow is installing a Teleport agent on an EC2 instance and connecting it to the cluster.
+Users who do not read the documentation and follow the instructions in the Teleport Web UI have an easy path forward
+that involves executing a single command.
+
+However, a user who attempts to read the documentation might follow a path that looks like this:
+
+1. Navigates to goteleport.com/docs
+2. Finds no information in the sidebar about installation or upgrading
```

Comment:
Worth noting that once PRs finally land, the plan is to eliminate the side nav on the homepage in lieu of tabs in a top nav, with a homepage that links out to key pages, including the installation guide, just below a feature on the "Get Started" guide:
<img width="1792" height="1188" alt="image" src="https://github.com/user-attachments/assets/c7b9d4dc-b9e5-4965-b2ab-e08811e5d86f" />